index.html

```diff
@@ -31,6 +31,9 @@
         <div class="mt-5">
           <p class="font-semibold font-30 text-white">Launching Soon!</p>
         </div>
+        <div class="mt-5">
+          <p class="font-18 text-white">In the meantime you can download our whitepaper <a href="whitepaper/unlock_opportunities.pdf">here</a>.</p>
```

Comment:
@sankarshanmukhopadhyay do we need to change the wording here? You want to merge it and then make a change in following PR? That way we know how it looks like.